src/Adapters/LaravelCacheAdapter.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Spatie\Prometheus\Adapters;
+
+use Illuminate\Contracts\Cache\Repository;
+use Prometheus\Counter;
+use Prometheus\Gauge;
+use Prometheus\Histogram;
+use Prometheus\Storage\InMemory;
+use Prometheus\Summary;
+
+class LaravelCacheAdapter extends InMemory
+{
+    private const CACHE_KEY_PREFIX = 'PROMETHEUS_';
+
+    private const CACHE_KEY_SUFFIX = '_METRICS';
+
+    private const STORES = [Gauge::TYPE, Counter::TYPE, Histogram::TYPE, Summary::TYPE];
```

Comment:
Refactor this to a regular array.

Reply:
In fact the rest of the const, can be refactored the regular protected properties as well.

Reply:
what do you mean by regular array?
as a property instead of a const?